ext/windows/run_facter_interactive.bat

```diff
@@ -0,0 +1,7 @@
+@echo Running facter on demand ...
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
+)
+elevate.exe "%~dp0facter_interactive.bat"

```

Comment:
Also, https://github.com/puppetlabs/puppet-agent/pull/515 will install elevate.exe and elevate.exe.config